csharp/src/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
```

Comment:
This file needs to be added to Makefile.am in the root directory. (That's what's causing the distcheck failures.)